docs/plugins/CreatedModifiedDate.md

```diff
@@ -13,6 +13,21 @@ This plugin accepts the following configuration options:
 
 - `priority`: The data sources to consult for date information. Highest priority first. Possible values are `"frontmatter"`, `"git"`, and `"filesystem"`. Defaults to `["frontmatter", "git", "filesystem"]`.
 
+If frontmatter is specified, the following frontmatter values are referenced:
+
+- created
+  - `created`
+  - `date`
+- modified
+  - `modified`
+  - `lastmod`
+  - `updated`
+  - `last-modified`
+- published
+  - `published`
+  - `publishDate`
+  - `date`
```

Comment:
I think a better wording for this is:

Quartz dealiased the following frontmatter value if the default does not exists (then you can link to the source of this dealias)

Fwiw can you also mention all of the field for that items that we dealias from frontmatter (probably we should add that in the frontmatter docs, then wikilinks to each of the relevant features, such as modified date)